tests/zfs-tests/tests/functional/history/history_006_neg.ksh

```diff
@@ -75,7 +75,8 @@ if ! is_linux; then
 	log_must zfs share $fs
 	log_must zfs unshare $fs
 fi
-log_must zfs send -i $snap1 $snap2 > /dev/null
+# https://github.com/openzfs/zfs/issues/11445
+log_must zfs send -i $snap1 $snap2 | cat > /dev/null
```

Comment:
That works.  If we wanted to be consistent we've handled this slightly differently elsewhere in the test suite.

```diff
diff --git a/tests/zfs-tests/tests/functional/history/history_006_neg.ksh b/tes>
index 19b7114fa..8782e9bb2 100755
--- a/tests/zfs-tests/tests/functional/history/history_006_neg.ksh
+++ b/tests/zfs-tests/tests/functional/history/history_006_neg.ksh
@@ -47,6 +47,8 @@ verify_runnable "global"

 function cleanup
 {
+       rm -f $TEST_BASE_DIR/devnull
        datasetexists $fs && destroy_dataset $fs -rf
        log_must zfs create $fs
 }
@@ -75,7 +77,7 @@ if ! is_linux; then
        log_must zfs share $fs
        log_must zfs unshare $fs
 fi
-log_must zfs send -i $snap1 $snap2 > /dev/null
+log_must zfs send -i $snap1 $snap2 > $TEST_BASE_DIR/devnull
 log_must zfs holds $snap1

 log_must eval "zpool history $TESTPOOL > $NEW_HISTORY"

```

Reply:
Is there a reason to prefer that over changing the other instance as well?

(Admittedly, my only reason for having a preference is "I've had to clean up /var/tmp/devnull files when the test suite crashed (on )a host".)

Reply:
Not a good one that I'm aware of.  I'm fine with either approach, and I admit I like yours better since we can't fail to cleanup.  The only downside is if we want to be consistent there are a fair number of existing tests which use `$TEST_BASE_DIR/devnull` which we'd have to switch.  Grep for `devnull` in the tests directory.

Reply:
I've never been one to shy away from enormous deltas.

Think I should do that as a followup or in this PR? I'm fine either way, just seems like something that could be taken care of as a followup step to unbreaking a test.

Reply:
Let's handle it in a follow up PR, I don't think we need to tackle it here.  Or necessarily even at all since it's such a minor thing.

Reply:
Heh, important absentmindedness I only realized when reworking every other instance - this will make this test step never fail, because log_must failme | true won't break without pipefail.

Oops.

Reply:
Ahh yes.  Now I recall, that would be the reason we went with the `> $TEST_BASE_DIR/devnull` approach originally.  Do you then want to update this PR to do the same?

Reply:
Nah, I just set pipefail. Something wrong with that?

Reply:
Nope as long as we're sure it works for `ksh`, it is covered in the man page so it should.

Reply:
I'm cleaning up a couple places it breaks expectations to set in _other_ tests, for a future PR, but for this one it should behave as expected with just the one `set -o pipefail`.

Reply:
Sounds good.  Then when this one finishes its CI runs I'll get it merged.